test/test_readers.py

```diff
@@ -30,7 +30,7 @@ def test_read(self):
         for index, x in enumerate(lines.split('\n')):
             self.assertEquals(x, str(index))
 
-    def test_read_n_chars(self):
+    def test_read_n_chars_1(self):
```

Comment:
there were 2 tests with the same name, separating them as `test_*_1` & `test_*_2`